wagtail/admin/widgets.py

```diff
@@ -71,11 +75,29 @@ def media(self):
 class AdminTimeInput(widgets.TimeInput):
     template_name = 'wagtailadmin/widgets/time_input.html'
 
-    def __init__(self, attrs=None, format='%H:%M'):
-        default_attrs = {'autocomplete': 'off'}
+    def __init__(self, attrs=None, format=None):
+        default_attrs = {'autocomplete': 'new-time'}
```

Comment:
autocomplete="off" is the current preferred attribute, as per 928e79ed3a119bf376a69b613517c6523100068b.

Reply:
Thanks for catching! Must have accidentally overridden this: I've changed back in the new fixup commit.